### PR TITLE
Add items table and model

### PR DESCRIPTION
**Pull request by tess-dev-main**
This PR adds Items to the database, updates the schema, and creates an Item model.

### CODE REVIEW

File: hangry_games/migrations/2024-09-16-210408_link_game_and_area_to_items/down.sql
Comment (tess-dev-main) [inline hangry_games/migrations/2024-09-16-210408_link_game_and_area_to_items/down.sql:1]: Oops! Noted.

Comment (tess-dev-main) [reply hangry_games/migrations/2024-09-16-210408_link_game_and_area_to_items/down.sql:1]: Will update.

File: hangry_games/migrations/2024-09-16-211214_add_durability_to_items/down.sql
Comment (tess-dev-main) [inline hangry_games/migrations/2024-09-16-211214_add_durability_to_items/down.sql:1]: Missed this one too.

File: hangry_games/migrations/2024-09-16-211214_add_durability_to_items/down.sql [hunk 1]
```diff
@@ -0,0 +1 @@
+-- This file should undo anything in `up.sql`
```
Comment (kennethlove) [inline hangry_games/migrations/2024-09-16-211214_add_durability_to_items/down.sql:1]: this down doesn't do anything

**Review by kennethlove — CHANGES_REQUESTED** [review]
Looks OK but I'd like to see some clean up. This only needs 1 migration so consolidate those down, please. Also make sure every up has a down and both of them have relevant, explanatory comments.

Other than that, seems like a fine PR.

File: hangry_games/migrations/2024-09-17-181252_add-items-consolidated/down.sql [hunk 2]
```diff
@@ -0,0 +1 @@
+DROP TABLE items;-- This file should undo anything in `up.sql`
```
Comment (kennethlove) [inline hangry_games/migrations/2024-09-17-181252_add-items-consolidated/down.sql:1]:
```suggestion
-- Removes the `items` table
DROP TABLE items;
```

File: hangry_games/migrations/2024-09-17-181252_add-items-consolidated/up.sql [hunk 3]
```diff
@@ -0,0 +1,23 @@
+-- Your SQL goes here
```
Comment (kennethlove) [inline hangry_games/migrations/2024-09-17-181252_add-items-consolidated/up.sql:1]:
```suggestion
-- Creates `items` table to store...
```

**Review by kennethlove — APPROVED** [review]
Looks good other than the creation comment needing to be filled out. Thanks!

**Pull request merged**